tests/components/tibber/test_services.py

```diff
@@ -246,6 +249,75 @@ async def test_get_prices_start_tomorrow(
     }
 
 
+@pytest.mark.parametrize(
+    "start_time",
+    [
+        STARTTIME.isoformat(),
+        STARTTIME.replace(tzinfo=None).isoformat(),
+        (STARTTIME + dt.timedelta(hours=4))
+        .replace(tzinfo=dt.timezone(dt.timedelta(hours=4)))
+        .isoformat(),
+    ],
+)
+async def test_get_prices_with_timezones(
+    freezer: FrozenDateTimeFactory,
+    start_time: str,
+) -> None:
+    """Test __get_prices with timezone and without."""
+    freezer.move_to(STARTTIME)
+    call = ServiceCall(DOMAIN, PRICE_SERVICE_NAME, {"start": start_time})
+
+    result = await __get_prices(call, hass=create_mock_hass())
```

Comment:
Please test the services by calling the services from the core service registry, while patching the library. Use `hass.services.async_call`.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations